src/modules/formReducer.ts

```diff
@@ -0,0 +1,5 @@
+const reducer = (state: any, { target }: any) => {
+  return { ...state, ...target }
+}
+
+export default reducer
```

Comment:
I couldn't find where it is used.
Could you share for the info for this?
Thanks!

Reply:
oh I'll be used for next PR(login form) I can delete from here and push next PR but if you think it's okay to leave it now, we can leave it here(as this PR is for initializing shared modules)
<img width="859" alt="Screen Shot 2021-08-25 at 9 29 38 AM" src="https://user-images.githubusercontent.com/49046928/130829413-1543ae1a-71de-4dcc-a4ed-58cf1a33207b.png">